extensions/smallrye-openapi/deployment/src/main/java/io/quarkus/smallrye/openapi/deployment/SmallRyeOpenApiProcessor.java

```diff
@@ -313,6 +314,15 @@ private OpenAPI generateAnnotationModel(IndexView indexView, Capabilities capabi
 
         String defaultPath = config.getValue("quarkus.http.root-path", String.class);
```

Comment:
I might be missing something, but is there any reason we are not using https://github.com/quarkusio/quarkus/blob/master/extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/HttpBuildTimeConfig.java#L15 ?

Reply:
It's the same config as far as I can see right ? Not sure what is better to use, I am easy with any.

Reply:
using the config root / object is future proof in case of renaming etc